File: Daily Questions/1653-minimum-deletions-to-make-string-balanced/1653-minimum-deletions-to-make-string-balanced.py
class Solution:
    def minimumDeletions(self, s: str) -> int:
        ans = count = 0
        for ch in s:
            if (ch == 'b'):
                count += 1
            elif (count > 0):
                ans += 1
                count -= 1
        return ans

        

File: 1653-minimum-deletions-to-make-string-balanced/1653-minimum-deletions-to-make-string-balanced.c
int minimumDeletions(char* s) {
    int ans = 0; 
    int count = 0;
    for(int i = 0; s[i] != '\0'; i++) {
        if (s[i] == 'b') {
            count++;
        }
        else if (count > 0) {
            ans++;
            count--;
        }
    }
    return ans;
}